doc/nrf/ug_thread_configuring.rst

```diff
@@ -304,13 +322,24 @@ The following table lists the supported features for each of these sets.
       - ✔
       -
 
-Replacing OpenThread Libraries
-******************************
+.. _thread_ug_feature_updating_libs:
+
+Updating OpenThread libraries
+=============================
+
+Each OpenThread sample provides a possibility to update nrfxlib's :ref:`nrfxlib:ot_libs` when the sample is configured to build OpenThread stack from source with :option:`CONFIG_OPENTHREAD_SOURCES`.
+You can use this functionality for certification with your own custom libraries.
 
-Each OpenThread example provides a possibility to replace libraries present in the nrfxlib when configured to build OpenThread stack from source with :option:`CONFIG_OPENTHREAD_SOURCES`.
-After the sample is build execute the following to overwrite the exising nrfxlib libraries with currently built ones::
+After a sample is built, run the following command to overwrite the update nrfxlib libraries to your latest version:
 
-  west build -t install_openthread_libraries
+.. parsed-literal::
+   :class: highlight
+
+   west build -t install_openthread_libraries
+
+.. note::
+    The libraries destination directory can differ.
+    When you selected :option:`CONFIG_OPENTHREAD_USER_CUSTOM_LIBRARY`, the location depends the chosen nRF Security backend, either :option:`CONFIG_CC310_BACKEND` or :option:`CONFIG_OBERON_BACKEND`.
```

Comment:
Something to consider: Maybe we should not mention backends here and mention it "depends on device hardware crypto capabilities" as it is more complex than just CC and OBERON i.e. on nRF53 we'll have CC312, and on 52833 currently we use mixed config oberon + vanilla due to lack of SHA224 support in oberon(to be added soon).